synapse/lib/storm.py

```diff
@@ -517,6 +517,9 @@ def getArgParser(self):
         pars.add_argument('--type', default=None, help='Re-index all properties of a specified type.')
         pars.add_argument('--subs', default=False, action='store_true', help='Re-parse and set sub props.')
         pars.add_argument('--form-counts', default=False, action='store_true', help='Re-calculate all form counts.')
+        pars.add_argument('--fire-handler', default=None, action='store', dest='fire_handler',
```

Comment:
default, action, dest parms are redundant here.

Reply:
habit of being explicit vs. impllicit with argparse